src/common/components/organisms/SetupLoadingScreen.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { SetupStep } from "@/common/data/stores/setup";
-import LoadingScreen from "../molecules/LoadingScreen";
+import LoadingScreen from "./LoadingScreen";
```

Comment:
nit: prefer absolute imports

Reply:
I just let the auto import generator do it, but I agree